buildSrc/src/main/java/io/micronaut/website/Utils.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package io.micronaut.website;
 
 import java.io.BufferedReader;
```

Comment:
Comment in wrong place because GitHub won't let me put it on `readFromInputStream`: I don't like much the fact that this method takes an inputstream as a parameter since it's unclear from the caller who is responsible for closing the stream.
Since you're always calling it with a resource from classpath, consider replacing it with a method which takes a path to a classpath resource instead.

Reply:
I have closed the input streams here https://github.com/micronaut-projects/micronaut-docs-index/pull/1/commits/332b2911e7518b456ee827d3a545702b11273ef8